04-HTML5-APIs/Ex01/script.js

```diff
@@ -0,0 +1,28 @@
+document.addEventListener('load', function() {
+		window.indexedDB = window.indexedDB || window.mozIndexedDB || window.webkitIndexedDB || window.msIndexedDB;
+		if (!window.indexedDB) {
+	 	window.alert("Your browser doesn't support a stable version of IndexedDB. Such and such feature will not be available.");
+		}
+		let request = window.indexedDB.open('TextDB', 3);
+		let db;
+		request.onsuccess = function(event) {
+	 	db = event.target.result;
+		};
+		let objectStore = db.createObjectStore('text');
+		document.getElementById('store').addEventListener('click', function() {
+			let transaction = db.transaction('text', 'readWrite');
+			transaction.oncomplete = function(event) {
+				objectStore.add(document.getElementById('textBox').value);
+			};
+		});
+		document.getElementById('clear').addEventListener('click', function() {
+			let transaction = db.transaction('text');
+			transaction.oncomplete = function(event) {
+				objectStore.clear();
+			}
+		});
+});
+
+document.getElementById('store').addEventListener('click', function() {localStorage.text = document.getElementById('textBox').value});
```

Comment:
Not sure if it's OK, but here's says both syntaxes are correct.
https://www.w3schools.com/html/html5_webstorage.asp